hive-tables-creation.py

```diff
@@ -1,6 +1,12 @@
 import boto3
 import logging
```

Comment:
i dont think you need to import logging in this script since you are using the logger you made in another module